coalaip/plugin.py

```diff
@@ -34,6 +34,32 @@ def generate_user(self, *args, **kwargs):
         """
 
     @abstractmethod
+    def get_history(self, persist_id):
+        """Get the ownership history of an entity on the persistence
+        layer.
+
+        Args:
+            persist_id (str): Id of the entity on the persistence layer
+
+        Returns:
+            list of dict: The ownership history of the entity. Each dict
+            is of the form::
+
+                {
+                    'user': A representation of a user as specified by the
+                            persistence layer (may omit secret details, e.g. private keys),
```

Comment:
I think for now, only having a single owner is enough.

In the future there must be a way for the pycoalaip-ledger plugin to recognize features of the ledger its persisting to. According to the features a specific ledger implements, persistence of models will have to be adjusted to the ledger at run-time. Ideally, every ledger implements a standardized feature-detection API.

For now: Lets have the simplest use case, which is single owner.

@msporny: Here you have a use cases for feature-detection built into blockchains. I think you were hinting on that in some of the talks we had and also in some of [your publications](https://github.com/WebOfTrustInfo/rebooting-the-web-of-trust-fall2016/blob/master/topics-and-advance-readings/fit-for-purpose-blockchains.md).